app/controllers/api/v1/study_files_controller.rb

```diff
@@ -469,7 +469,8 @@ def parse
             end
           when 'Expression Matrix'
             @study_file.update(parse_status: 'parsing')
```

Comment:
should we feature-flag this?  Obviously, it would be a flag that is only used at the server level, not users/branding groups.  But if expression parsing via ingest pipeline goes south, it would be nice to revert without a release.  Not sure the risk of it going south is large enough to be worth the hassle/complexity of a feature flag though

Reply:
My gut says no - I'm not 100% sure the feature flagging framework will work with this, and the complexity of doing that would be such that I would worry that will have its own bugs/issues.  Our existing test infrastructure has shown that dense parsing via ingest pipeline works, and further manual spot checks have shown that the plots before and after are identical.  I even checked the databased for every gene parsed from one of our test matrices and the scores are perfectly aligned before and after.

If we want to have an "escape lever", we should simply have a PR ready to merge that would undo the integration.  We wouldn't even necessarily need to roll back the Docker image - just the controller logic for calling ingest.  But we have ~8 full business days to test this on staging on this sufficiently.  I don't think feature flagging this is the right way to go.